requirements.txt

```diff
@@ -1,5 +1,6 @@
 # app requirements
 CairoSVG==2.2.1
+celery==3.1.26.post2 # pyup: <4
```

Comment:
celery 3 😢

can we just quietly put this on celery 4? does that work? (i know that 3.1.26 and 4 were cross-compatible when i tried)

Reply:
Changed to Celery 4 in d716f29. It worked fine locally, but I guess we may want to do a load test on staging when this is deployed.